src/site/content/en/payments/registering-a-web-based-payment-app/index.md

```diff
@@ -0,0 +1,321 @@
+        <td>
+          A supported payment method identifier.
+        </td>
+      </tr>
+      <tr>
+        <td>
+          <code>capabilities</code>
+        </td>
+        <td>
+          Sets the <a href="https://w3c.github.io/payment-handler/#capabilities-example">payment method specific parameters as an object</a>. As of July 2020, basic-card is the only payment method that accepts capabilities.
+        </td>
+      </tr>
+    </tbody>
+  </table>
+</div>
+
+{% Aside %}
+Chrome ignores the `name` and `icons` properties. It respects the [web app
+manifest](/setting-up-a-payment-method/#step-3:-serve-a-web-app-manifest)'s
+respective properties instead, but other browsers may behave differently.
+{% endAside %}
+
+### Enable delegations
+
+Entering shipping address and contact information through a web form can be a
+cumbersome experience for customers. It can cause errors and lower conversion
+rate.
+
+That's why the Payment Request API supports a feature to request shipping
+address and contact information. This provides multiple benefits:
+
+* Users can pick the right address with just a few taps.
+* The address is always returned in [the standardized
+  format](https://w3c.github.io/payment-request/#paymentaddress-interface).
+* Submitting an incorrect address is less likely.
+
+Payment apps can integrate this functionality to offer a more unified payment
+experience and this is called "delegation".
+
+<figure class="w-figure" style="width:300px; margin:auto;">
+  <video controls autoplay loop muted class="w-screenshot">
+    <source src="https://storage.googleapis.com/web-dev-assets/payments/skip-the-sheet.webm" type="video/webm">
+    <source src="https://storage.googleapis.com/web-dev-assets/payments/skip-the-sheet.mp4" type="video/mp4">
+  </video>
+  <figcaption class="w-figcaption">
+    Checkout flow with a web-based payment app.
+  </figcaption>
+</figure>
+
+Web-based payment apps can declare their support for delegation using the web
+app manifest's `payment.supported_delegations` field or through JavaScript.
+
+To let the browser know that the payment app accepts a delegation, use
+[`.enableDelegations()`](https://w3c.github.io/payment-handler/#enabledelegations-method).
+
+{% Label %}[payment handler] payment app's landing page{% endLabel %}
+
+```js
+...
+ await registration.paymentManager.enableDelegations([
+  'shippingAddress', 'payerName'
+]);
+...
+```
+
+You can declare supported delegation options as an array of strings:
+
+* `shippingAddress`: The payment handler can provide a shipping address.
+* `payerName`: The payment handler can provide the payer's name.
+* `payerPhone`: The payment handler can provide the payer's phone number.
+* `payerEmail`: The payment handler can provide the payer's address.
+
+## Debugging a web-based payment app
+
+When developing a web-based payment app frontend, you'll probably jump between
+merchant perspective and payment app perspective. The following debugging tips
```

Comment:
Perspective?

Reply:
The merchant and the payment app are two separate web contexts the developer has to deal with. If there' any better wording on this, please let me know.

Reply:
`context`?